_episodes/04-quality-control-2.md

```diff
@@ -0,0 +1,167 @@
+keyboard) should never be entered. Likewise if one of the kinds of data being
+collected is a count, only integers greater than or equal to zero should be
+allowed.
+
+To control the kind of data entered into a a spreadsheet we use Data Validation
+(Excel) or Validity (Libre Office Calc), to set the values that can be entered
+in each data column.
+
+1\. Select the cells or column you want to validate
+
+2\. On the `Data` tab select `Data Validation`
+
+![Image of Data Validation button on Data tab](../fig/data_validation.png)
+
+3\. In the `Allow` box select the kind of data that should be in the
+   column. Options include whole numbers, decimals, lists of items, dates, and
+   other values.
+
+![Image of Data Validation window](../fig/data_validation_window.png)
+   
+4\. After selecting an item enter any additional details. For example, if you've
+   chosen a list of values, enter a comma-delimited list of allowable
+   values in the `Source` box.
+
+Let's try this out by setting the plot column in our spreadsheet to only allow
+plot values that are integers between 1 and 24.
+
+1. Select the `plot_id` column
+2. On the `Data` tab select `Data Validation`
+3. In the `Allow` box select `Whole number`
+4. Set the minimum and maximum values to 1 and 24.
+
+![Image of Data Validation window for validating plot values](../fig/plot_validation.png)
+
+Now let's try entering a new value in the plot column that isn't a valid
+plot. The spreadsheet stops us from entering the wrong value and asks us if we
+would like to try again.
+
+![Image of error when trying to enter invalid data](../fig/invalid_value.png)
+
+You can also customize the resulting message to be more informative by entering
+your own message in the `Input Message` tab
+
+![Image of Input Message tab](../fig/input_message.png)
+
+or allow invalid data to result in a warning rather than an error by modifying the `Style`
+option on the `Error Alert` tab.
+
+![Image of Error Alert tab](../fig/error_alert.png)
+
+Quality assurance can make data entry easier as well as more robust. For
+example, if you use a list of options to restrict data entry, the spreadsheet
+will provide you with a drop-downlist of the available items. So, instead of
+trying to remember how to spell *Dipodomys spectabilis*, you can select the
+right option from the list.
+
+![Image of drop-down menu](../fig/drop_down_list.png)
+
+# Quality Control
+
+Tip: *Before doing any quality control operations, save your original file with the formulas and a name indicating it is the original
+data. Create a separate file with appropriate naming and versioning, and ensure your data is stored as values and not as formulas. 
+Because formulas refer to other cells, and you may be moving cells around, you may compromise the integrity of your data if you do not
+take this step!*
+
+readMe (README) files: As you start manipulating your data files, create a readMe document / text file to keep track of your files and
```

Comment:
use only README (it's the common practice)

Reply:
Thanks Francois. I'll make this change when I put in the PR for this episode.